index.ts

```diff
@@ -1379,6 +1394,149 @@ async function submitMtpV2ZkResponse(useMongoStore = false) {
   console.log('Balance after', await erc20Airdrop.balanceOf(await ethSigner.getAddress()));
 }
 
+async function submitV3ZkResponse(useMongoStore = false) {
+  let dataStorage, credentialWallet, identityWallet;
+  if (useMongoStore) {
+    ({ dataStorage, credentialWallet, identityWallet } = await initMongoDataStorageAndWallets(
+      defaultNetworkConnection
+    ));
+  } else {
+    ({ dataStorage, credentialWallet, identityWallet } = await initInMemoryDataStorageAndWallets(
+      defaultNetworkConnection
+    ));
+  }
+
```

Comment:
I am not sure about the project but we could separate the responsabilities in some functions/objects.